subset/security/brute_test/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```

Comment:
Not sure we need your eclipse preferences in the repo

Reply:
Ed's switch test is a good model for this:

https://github.com/arupiot/daq/tree/master/subset/switches

The only things in the directory are:
- java source
- Dockerfile
- Shell script to run the test
- gitignore
- build.conf

Reply:
Removed a majority of what I believe are junk files in the project, still keeping the gradle files (for now), until I run tests to see if they are needed at runtime, Although me and Ed have tests in java, mine is in Gradle while he has used Maven so the files needed will 100% similar, (barring jar files).